packages/esm-patient-registration-app/src/config-schema.ts

```diff
@@ -263,6 +301,107 @@ export const esmPatientRegistrationSchema = {
       ],
       _description: 'Provide ability to configure sex options.',
     },
+
+    maritalStatus: {
+      _type: Type.Array,
+      _elements: {
+        value: {
+          _type: Type.String,
+          _description: 'The value for sex option',
```

Comment:
I don't think this should be `the value for sex option` 🤔, I think it should probably be `the value for marital status.`